src/serverMiddleware/routes.js

```diff
@@ -7,6 +7,7 @@
 
 module.exports = [
 	`^/administration/datasources?`,
+	`^/administration/students?`,
```

Comment:
Let's not commit this at the current situation where we release often. Otherwise we may break the system with our unfinished page.
```suggestion
	// `^/administration/students?`,
```